Cisco/cisco-esa/_meta/smart-descriptions.json

```
@@ -0,0 +1,291 @@
+[
+    {
+        "value": "{source.ip} connected to {destination.ip} on port {destination.port}.",
+        "conditions": [
+            {
+                "field": "source.ip"
+            },
+            {
+                "field": "destination.ip"
+            },
+            {
+                "field": "destination.port"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "source.ip",
+                "target": "destination.ip",
+                "type": "connected to"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was delivered",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "DELIVERED"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was delivered"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was delivered",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "delivered"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was delivered"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was quarantined",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "QUARANTINED"
```

With the suggestion to lower `event.action`:
```suggestion
                "value": "quarantined"
```